app-drawer/app-drawer.html

```diff
@@ -303,22 +319,24 @@
         this._updateDocScroll();
       },
 
-      _positionChanged: function() {
-        this.fire('app-drawer-position-changed');
-      },
-
       _translateDrawer: function(x) {
         this.transform('translate3d(' + x + 'px, 0, 0)', this.$.contentContainer);
       },
 
       _updateDocScroll: function() {
-        this.shouldEnableDocScroll(this.persistent || !this.opened);
+        var disableScroll = !this.persistent && this.opened;
+        document.body.style.overflow = disableScroll ? 'hidden' : '';
+
+        var appHeaderLayout = document.querySelector('app-header-layout');
```

Comment:
Yea, I think that'd be better too. So essentially the functionality will be, "When `app-drawer` is opened, we will disable document scrolling on desktop/Android. Additionally, if you use `app-header-layout` inside `app-drawer-layout`, we will disable `app-header-layout`'s scrolling for iOS when `app-drawer` is opened."